v0/destinations/ga4/utils.js

```diff
@@ -17,13 +17,54 @@ const { TRANSFORMER_METRIC } = require("../../util/constant");
 const ErrorBuilder = require("../../util/error");
 const { mappingConfig, ConfigCategory } = require("./config");
 
+/**
+ * get time difference in hours
+ * @param {*} date1
+ * @param {*} date2
+ * @returns
+ */
+function getDifferenceInHours(date1, date2) {
+  const diffInMs = date2 - date1;
+  return diffInMs / (1000 * 60 * 60);
+}
+
+/**
+ * get time difference in minutes
+ * @param {*} date1
+ * @param {*} date2
+ * @returns
+ */
+function getDifferenceInMinutes(date1, date2) {
+  const diffInMs = date2 - date1;
+  return diffInMs / (1000 * 60);
+}
+
 /**
  * converts to Unix timestamp (in microseconds)
  * @param {*} timestamp
  * @returns
  */
 function msUnixTimestamp(timestamp) {
+  const currentTime = new Date();
```

Comment:
are both of these in unix format?

Reply:
How are we confirming there wont be any offset because of timezone

Reply:
To handle this we are converting and checking the timestamp_micros in local time. The following L:49 does that.

Reply:
Lets do the diff in unix

Reply:
Done